### ISSUE TITLE
Aqara Smart Video Doorbell G4

### ISSUE THREAD

**Issue by BaconSandwichMan** [jghaanstra/com.xiaomi-miio#238]
Is it possible to add this to the app ?

Aqara Smart Video Doorbell G4
https://www.aqara.com/eu/product/smart-video-doorbell-g4/

**Comment by jghaanstra** [comment]
No, this app is for Xiaomi WiFi devices that support the MIoT protocol. It's unclear to me if Aqara WiFi products allow for any form of third integration without using their own cloud.
